docs: add documentation for hasBinaryFile built-in function (#126)
reviewpad/docs: docs/guides/built-ins/functions/hasBinaryFile/index.mdx

import CodeBlock from "@theme/CodeBlock";
import Example from "!!raw-loader!./example.yml";

**Description**:

> Checks if a pull request adds or changes a binary file.

Reviewer: ```suggestion
Checks if a pull request has a binary file in the patch.
```

**Parameters**:

_none_

**Return value**:

| Type      | Description                                                                  |
| :-------- | :--------------------------------------------------------------------------- |

> | `boolean` | `true` if the pull request adds or changes a binary file, `false` otherwise. |

Reviewer: ```suggestion
| `boolean` | `true` if the pull request contains a binary file, `false` otherwise. |
```
